src/main/java/org/assertj/guava/api/RangeSetAssert.java

```diff
@@ -0,0 +1,982 @@
+   * @return this {@link RangeSetAssert} for assertions chaining.
+   * @throws AssertionError if the actual {@code RangeSet} is {@code null}.
+   * @throws AssertionError if the actual {@code RangeSet} does not enclose at least one range from the given range set.
+   * @throws IllegalArgumentException if range set is null or it is empty while actual is not empty.
+   */
+  public RangeSetAssert<T> enclosesAnyElementsOf(final RangeSet<T> rangeSet) {
+    rangeSets.assertEnclosesAnyElementsOf(info, actual, rangeSet);
+    return myself;
+  }
+
+  /**
+   * Verifies that the given {@code RangeSet} does not enclose the given ranges.
+   * <p>
+   * Example:
+   *
+   * <pre><code class='java'>
+   * RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create();
+   *
+   * rangeSet.add(Range.closed(0, 100));
+   * rangeSet.add(Range.closed(200, 300));
+   * rangeSet.add(Range.closed(500, 600));
+   *
+   * assertThat(rangeSet).doesNotEnclose(Range.closed(-10, 10),
+   *                                     Range.open(150, 160),
+   *                                     Range.open(590, 700));</code></pre>
+   * </p>
+   *
+   * @param ranges the ranges to check whether the actual {@code RangeSet} does not enclose them.
+   * @return this {@link RangeSetAssert} for assertions chaining.
+   * @throws AssertionError if the actual {@code RangeSet} is {@code null}.
+   * @throws AssertionError if the actual {@code RangeSet} encloses any of the given ranges.
+   * @throws IllegalArgumentException if ranges are null or ranges are empty while actual is not empty.
+   */
+  public RangeSetAssert<T> doesNotEnclose(final Range<T>... ranges) {
+    rangeSets.assertDoesNotEnclose(info, actual, ranges);
+    return myself;
+  }
+
+  /**
+   * Verifies that the given {@code RangeSet} does not enclose any of the given ranges.
+   * <p>
+   * Example:
+   *
+   * <pre><code class='java'>
+   * RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create();
+   *
+   * rangeSet.add(Range.closed(0, 100));
+   * rangeSet.add(Range.closed(200, 300));
+   * rangeSet.add(Range.closed(500, 600));
+   *
+   * assertThat(rangeSet).doesNotEncloseAnyElementsOf(Arrays.asList(Range.closed(-10, 10),
+   *                                                                Range.open(150, 160),
+   *                                                                Range.open(590, 700));</code></pre>
+   * </p>
+   *
+   * @param ranges the ranges to check whether the actual {@code RangeSet} does not enclose any of them.
+   * @return this {@link RangeSetAssert} for assertions chaining.
+   * @throws AssertionError if the actual {@code RangeSet} is {@code null}.
+   * @throws AssertionError if the actual {@code RangeSet} encloses any of the given ranges.
+   * @throws IllegalArgumentException if ranges are null or ranges are empty while actual is not empty.
+   */
+  public RangeSetAssert<T> doesNotEncloseAll(final Iterable<Range<T>> ranges) {
```

Comment:
rename `doesNotEncloseAll` -> `doesNotEncloseAnyRangesOf`

Reply:
applies to other assertions too

Reply:
done